assets/src/edit-story/components/helpCenter/navigator/stories/index.js

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import { useState } from 'react';
+import styled, { ThemeProvider } from 'styled-components';
+/**
+ * Internal dependencies
+ */
+import { Navigator as HelpCenterNavigator } from '..';
+import { theme as dsTheme, ThemeGlobals } from '../../../../../design-system';
+
+export default {
+  title: 'Stories Editor/Components/Help Center/Navigator',
+};
+
+const Bg = styled.div`
+  position: relative;
+  top: 0;
+  left: 0;
+  height: 80vh;
```

Comment:
would it be worthwhile to add min/max height to control for edge case viewport sizes?

Reply:
I'll just make it the whole viewport. It's pretty layout specific, so it's fine if it doesn't always look great in storybook.